react/o11y/extraArgs.ts

```diff
@@ -0,0 +1,94 @@
+interface RuntimeInfo {
+  account: string | null
+  workspace: string | null
+  route: { path: string | null; blockId: string | null }
+  culture: { locale: string | null }
+  production: boolean | null
+  loadedDevices: Array<string | null>
+}
+
+/**
+ * Infer the runtime information from the window.location object.
+ */
+export function inferRuntimeInfo(): Partial<RuntimeInfo> {
+  const { host = '' } = window.location
+  let account = ''
+  let workspace = ''
+
+  if (!host.includes('--')) {
+    account = host.split('.')[0]
+    workspace = 'master'
+  } else {
+    workspace = host.split('--')[0]
+    account = host.split('--')[1].split('.')[0]
+  }
+
+  const { pathname } = window.location
+
+  const locale =
+    navigator?.language ?? Intl.DateTimeFormat().resolvedOptions().locale
+
+  return {
+    account,
+    workspace,
+    route: { path: pathname, blockId: null },
+    culture: { locale },
+  }
+}
+
+/**
+ * Gets the Runtime information from the global __RUNTIME__ object
+ * injected by Render Server, if available, otherwise infer the runtime
+ * information from the window.location.
+ */
```

Comment:
```suggestion
/**
 * Gets the Runtime information from the global __RUNTIME__ object
 * injected by Render Server, if available, otherwise infer the runtime
 * information from the window.location.
 * 
 * WARNING: Render server will load the render-runtime error.tsx file based on the appId version (see render-server template.ts error() <head> tag). Since the runtime.appId is not reliably set, a fallback version of vtex.render-runtime is hardcoded in render-server error.ts errorMiddleware() function. Because of this, updates in the structure of the app below might need an update to this fallback version along with a new render-server deployment
 */
```

Galera, acham que vale a pena?

Reply:
@vmourac-vtex vou incluir esse comentário no próprio ErrorPage component, ok?

Reply:
Feito!